.github/workflows/rust.yml

```diff
@@ -79,9 +75,16 @@ jobs:
 
       - name: Cache Cargo registry
         uses: Swatinem/rust-cache@v2
+        with:
+          save-if: false
 
       - name: Run clippy
-        run: cargo clippy --no-deps -- -D warnings
+        uses: giraffate/clippy-action@v1
```

Comment:
This looks cool!

Reply:
Shamelessly stolen from Gjengset https://github.com/jonhoo/rust-ci-conf/blob/main/.github/workflows/check.yml#L57-L61